code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -510,6 +510,10 @@ obj/effect/sweatsplash/proc/splash()
 									organcantidates += O
 								continue
 							organcantidates += O
+							if(ishuman(M))
+								var/mob/living/carbon/human/H = M //To view species
+								if(!is_species(H, /datum/species/plasmaman))
+									O -= /obj/item/organ/lungs/plasmaman //So this disease doesn't eventually kill everyone with lungs
```

Comment:
Hmm, wouldn't it be better to just not have the disease add them in the first place?

Reply:
That's what it's doing. Its removing the incorrect lungs from the list of possible organs

Reply:
Additionally taking that into the loop means those checks are running every single time, whereas this just nukes them from it once.